src/lib/test/InfoTests.cpp

```diff
@@ -207,28 +173,33 @@ void InfoTests::testGetSlotInfoAlt()
 	// (Re)initialize the token
 	rv = C_Initialize(NULL_PTR);
 	CPPUNIT_ASSERT(rv == CKR_OK);
-	rv = C_InitToken(SLOT_INIT_TOKEN, pin, pinLength, label);
+	rv = C_InitToken(m_initializedTokenSlotID, m_soPin1, m_soPin1Length, label);
 	CPPUNIT_ASSERT(rv == CKR_OK);
 	C_Finalize(NULL_PTR);
 
-	rv = C_GetSlotInfo(SLOT_NO_INIT_TOKEN, &slotInfo);
+	rv = C_GetSlotInfo(m_notInitializedTokenSlotID, &slotInfo);
 	CPPUNIT_ASSERT(rv == CKR_CRYPTOKI_NOT_INITIALIZED);
 
 	rv = C_Initialize(NULL_PTR);
 	CPPUNIT_ASSERT(rv == CKR_OK);
 
-	rv = C_GetSlotInfo(SLOT_NO_INIT_TOKEN, NULL_PTR);
+	rv = C_GetSlotInfo(m_notInitializedTokenSlotID, NULL_PTR);
 	CPPUNIT_ASSERT(rv == CKR_ARGUMENTS_BAD);
 
-	rv = C_GetSlotInfo(SLOT_INVALID, &slotInfo);
+	rv = C_GetSlotInfo(m_invalidSlotID, &slotInfo);
 	CPPUNIT_ASSERT(rv == CKR_SLOT_ID_INVALID);
 
-	rv = C_GetSlotInfo(SLOT_NO_INIT_TOKEN, &slotInfo);
+	rv = C_GetSlotInfo(m_notInitializedTokenSlotID, &slotInfo);
 	CPPUNIT_ASSERT(rv == CKR_OK);
 	CPPUNIT_ASSERT((slotInfo.flags & CKF_TOKEN_PRESENT ) == CKF_TOKEN_PRESENT);
 	CPPUNIT_ASSERT((slotInfo.flags & CKF_REMOVABLE_DEVICE ) == CKF_REMOVABLE_DEVICE);
 
 	C_Finalize(NULL_PTR);
+#ifndef _WIN32
```

Comment:
Just guessing that this was a copy-paste error. Or do we need to set the environment variables here?